web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.styles.js

```diff
@@ -0,0 +1,37 @@
+import theme from "common/theme";
+
+const whatIsWcsStyles = {
+  whatIsWcsStylesContainer: {
+    padding: "24px",
+    maxWidth: "1152px", // to match Upcoming Events. Halp me here plz
```

Comment:
https://github.com/doyourownswing/doyourownswing.github.io/blob/4dbd37544e16516f7d6eca458e6cc535dca4aa6c/web/src/components/home/subcomponents/upcoming_events/upcoming_events.js#L80C8-L80C17

Use Container, inside a Box (Box is the whole section element, and Container holds the content and has the maxWidth styling)

Reply:
thanks - updated